minio.yaml

```diff
@@ -0,0 +1,88 @@
+kind: Service
```

Comment:
why do we need this file?

Reply:
we need this for development purposes. For testing the workloads with directpv storage class.

Reply:
Add a functional tests for that and remove this file.